lib/fluent/plugin/out_sumologic.rb

```diff
@@ -182,15 +182,15 @@ def formatted_to_msgpack_binary
     true
   end
 
-  def sumo_key(sumo_metadata, record, tag)
-    source_name = sumo_metadata['source'] || @source_name
-    source_name = expand_param(source_name, tag, nil, record)
+  def sumo_key(sumo_metadata, tag, chunk)
```

Comment:
is there a good reason to revert the order of `tag`/`record/chunk`?

Reply:
actually tag isn't needed here as it can be retrieved from `chunk.metadata`